pallets/risk-manager/src/lib.rs

```diff
@@ -613,24 +614,38 @@ impl<T: Config> Pallet<T> {
 		liquidated_pool_id: CurrencyId,
 		total_repay_amount: Balance,
 		is_partial_liquidation: bool,
-	) -> result::Result<(Balance, Balance, Balance), DispatchError> {
+	) -> result::Result<(Balance, Balance, Balance, bool), DispatchError> {
 		let liquidation_fee = Self::risk_manager_dates(liquidated_pool_id).liquidation_fee;
+		let price_borrowed =
+			T::PriceSource::get_underlying_price(liquidated_pool_id).ok_or(Error::<T>::InvalidFeedPrice)?;
 
 		let temporary_factor = match is_partial_liquidation {
 			true => Rate::saturating_from_rational(30, 100),
 			false => Rate::one(),
 		};
 
-		// seize_amount = liquidation_fee * temporary_factor * total_repay_amount
-		let seize_amount = Rate::from_inner(total_repay_amount)
+		// repay_amount = temporary_factor * total_repay_amount
+		let mut repay_amount = Rate::from_inner(total_repay_amount)
 			.checked_mul(&temporary_factor)
-			.and_then(|v| v.checked_mul(&liquidation_fee))
 			.map(|x| x.into_inner())
 			.ok_or(Error::<T>::NumOverflow)?;
 
-		// repay_amount = temporary_factor * total_repay_amount
-		let repay_amount = Rate::from_inner(total_repay_amount)
-			.checked_mul(&temporary_factor)
+		let liquidation_pool_balance = T::LiquidationPoolsManager::get_pool_available_liquidity(liquidated_pool_id);
+		let liquidation_pool_balance_usd = Rate::from_inner(liquidation_pool_balance)
+			.checked_mul(&price_borrowed)
+			.map(|x| x.into_inner())
+			.ok_or(Error::<T>::NumOverflow)?;
+
+		// If there is not enough liquidity in the liquidation pool, then we do not change
+		// the user's liquidation attempts counter.
+		let is_need_mutate_attempts = liquidation_pool_balance_usd >= repay_amount;
```

Comment:
```suggestion
		let increment_attempt_counter = liquidation_pool_balance_usd >= repay_amount;
```
Guess is_ prefix is not always suitable. is_empty is ok, is_increment_attempt_counter_required is good, but long.

Reply:
Fixed